Library/Homebrew/github_packages.rb

```diff
@@ -235,10 +232,18 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:)
       end
       raise TypeError, "unknown tab['built_on']['os']: #{tab["built_on"]["os"]}" if os.blank?
 
-      os_version = if tab["built_on"].present? && tab["built_on"]["os_version"].present?
-        tab["built_on"]["os_version"]
+      os_version = tab["built_on"]["os_version"] if tab["built_on"].present?
+      os_version = case os
+      when "darwin"
+        os_version || "macOS #{MacOS::Version.from_symbol(bottle_tag)}"
+      when "linux"
+        (os_version || "Ubuntu 16.04.7").delete_suffix " LTS"
```

Comment:
@sjackman Can this `Ubuntu 16.04.7` here be cleaned up to use a constant instead?

Reply:
@sjackman Also the `.7` here definitely seems too specific.

Reply:
Opened PR https://github.com/Homebrew/brew/pull/11035